Policies/github-terms-of-service.md

```diff
@@ -36,15 +36,24 @@ Read this Terms of Service agreement carefully before accessing or using GitHub.
 | [Q. Changes to these Terms of Service](#q-changes-to-these-terms) | We may modify this agreement, but we will give you 30 days' notice of changes that affect your rights. |
 | [R. Miscellaneous](#r-miscellaneous) | Please see this section for legal details including our choice of law. |
 
+</details>
+
 ## The GitHub Terms of Service
 Effective date: November 13, 2019
 
 ### Section Headings and Summaries
-Throughout this Agreement, each section includes titles and sometimes brief summaries of the following terms and conditions. These section titles and brief summaries (summaries are preceded by the text 'Short version:') are not legally binding—you can completely ignore the summaries to reduce your reading. 
+The different sections in this Agreement have titles and sometimes also brief summaries. Such titles and brief summaries are not legally binding. Each such summary takes the form of a collapsible text that you can expand or collapse by clicking upon the summary's heading, where the summary's heading is always the text 'Click for short version'. Each summary appears just after the relevant section title, and immediately before the relevant section body. You can completely ignore such summaries. 
 
 ### A. Definitions
+<details><summary open="false"><b>Click for short version</b> </summary>
```

Comment:
:-1:

Reply:
Because you haven't commented on the other 'short versions' contractions, I'm wondering whether you made a mistake in down-voting this change...